src/storage_broker/app.py

```diff
@@ -107,7 +107,9 @@ def main():
 
         metrics.message_consume_count.inc()
         if msg.topic() == config.EGRESS_TOPIC:
-            announce(decoded_msg)
+            headers = dict((x, y) for (x, y) in msg.headers())
```

Comment:
Can you do
```
dict(msg.headers())
```

Reply:
I guess I can. I thought I had tried that. Update in coming